src/CppParser/Parser.cpp

```diff
@@ -63,6 +63,25 @@ using namespace CppSharp::CppParser;
 // We use this as a placeholder for pointer values that should be ignored.
 void* IgnorePtr = (void*) 0x1;
 
+bool IsNamespaceStd(const clang::DeclContext* Namespace)
+{
+    if (!Namespace)
```

Comment:
Is there any possibility to `Namespace` actually being null? If not I'd rather put an `assert` here.